client/bgpPeerP_service.go

```diff
@@ -98,3 +99,83 @@ func (sm *ServiceManager) ReadRelationbgpRsPeerPfxPolFromBgpPeerConnectivityProf
 	}
 
 }
+
+func (sm *ServiceManager) CreateRelationbgpRsPeerToProfile(parentDn, annotation, direction string, tDn string) error {
+	dn := fmt.Sprintf("%s/rspeerToProfile-[%s]-%s", parentDn, tDn, direction)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"annotation": "%s",
+				"tDn": "%s",
+				"direction": "%s"
+			}
+		}
+	}`, "bgpRsPeerToProfile", dn, annotation, tDn, direction))
+
+	attributes := map[string]interface{}{
+		"direction": direction,
+	}
+
+	var output map[string]interface{}
+	err_output := json.Unmarshal([]byte(containerJSON), &output)
+
+	if err_output != nil {
+		return err_output
+	}
+	for _, value := range output {
+		if rec, ok := value.(map[string]interface{}); ok {
```

Comment:
what is rec? Can you put a better name for it?

Reply:
rec is a map of all the values of bgpRsPeerToProfile
It is not required here, so removing it.
Will use a better name if used in other resources.